specification/monitor/resource-manager/readme.md

```diff
@@ -31,6 +31,15 @@ openapi-type: arm
 tag: package-2020-10
```

Comment:
Question, are you updating the default tag?

Reply:
No, I'm not